src/app/pages/send/send-crypto-asset-form/form/btc/use-btc-choose-fee.tsx

```diff
@@ -28,6 +28,7 @@ export function useBtcChooseFee() {
   return {
     amountAsMoney,
     onGoBack() {
+      // FIXME 4370 onGoBack - need to refactor this to not be baked into onGoBack logic
```

Comment:
@fbwoolf - if you have time can you help me think about how to make sure this still works? 

With the new headers I am setting `onGoBack` in the main [`container`](https://github.com/leather-wallet/extension/pull/4655/files#diff-550189a0c8f9f6f24b0af91a7e6c714c978d2935b94239cedb39bf11a23532e1).

Doing that means I can share it more as it's usually just `navigate(-1)` but in some cases like this, before we go back we seem to do some cleaning up like here where we `setSelectedFeeType`

I'd like to move this logic to `src/app/pages/send/send-crypto-asset-form/form/btc/btc-choose-fee.tsx` if possible

Reply:
We do a very similar thing in `src/app/pages/send/send-crypto-asset-form/form/brc-20/brc-20-choose-fee.tsx` but there we set the `setSelectedFeeType` to `null`.

I wonder if we could instead reset the value on component unmount? Or else always default the value so it shows what we want on first load?

Reply:
yeah, reset on unmount is also ok

Reply:
I did some code investigations here and decided to just get rid of this altogether. I checked and it seems like its OK: 
https://github.com/leather-wallet/extension/assets/2938440/947ba5fe-9db7-413b-9ec8-4e7138bdc86d

If this is breaking something it will be a `P4` bug so I can fix it later. 

I was going to add some cleanup logic but I can't find anywhere the value is initially set anyway so I am going to just remove it as per [documentation here](https://react.dev/reference/react/useEffect#my-cleanup-logic-runs-even-though-my-component-didnt-unmount)

Reply:
Lgtm 👍